FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/SchemaConfiguration.cs

```diff
@@ -17,15 +17,15 @@ public class SchemaConfiguration
         public string SchemaCollectionDirectory { get; set; } = ConfigurationConstants.DefaultSchemaDirectory;
 
         /// <summary>
-        /// Gets bool value for whether enable customized schema.
+        /// Gets or sets bool value for whether enable customized schema.
         /// </summary>
         [JsonProperty("enableCustomizedSchema")]
-        public bool EnableCustomizedSchema { get; } = false;
+        public bool EnableCustomizedSchema { get; set; } = false;
 
         /// <summary>
         /// Gets Azure Container Registry image reference of customized schema templates.
         /// </summary>
-        [JsonProperty("customizedSchemaImageReference")]
-        public string CustomizedSchemaImageReference { get; } = string.Empty;
+        [JsonProperty("schemaImageReference")]
+        public string SchemaImageReference { get; set; } = string.Empty;
```

Comment:
Will only one of the two configuration `SchemaCollectionDirectory` and `SchemaImageReference` take effect?
If yes, how about merge these two configuration?

Reply:
Sure, updated.